sourcegraph/about
# Missing special blogposts & redirects

This closes #5489 & #5490. Audits any missing special blogposts & sets up redirects for old special blogposts urls.

### Changelog
- 1 blogpost had a typo in it's url (was present prior to the Re-Platform merge too). Typo is fixed
- Redirects setup for special blogposts & old special blogposts indices => the default blog index.

### Note
- No other blogposts were missing

### Test
1. Ensure prettier has standardized the proposed changes.
2. Ensure go blogpost index (`/go`) redirects to `/blog`
- Ensure old go blog posts redirect to their new url (i.e. `/go/my-journey-to-go`)
3. Ensure strange-loop blogpost index (`/strange-loop`) redirects to `/blog`
- Ensure old strange-loop blog posts redirect to their new url (i.e. `/strange-loop/strange-loop-2019-css-algorithms`)
4. Ensure graphql blogpost index (`/graphql`) redirects to `/blog`
- Ensure old graphql blog posts redirect to their new url (i.e. `/graphql/teaching-graphql`)
5. Ensure github-universe blogpost index (`/github-universe`) redirects to `/blog`
- Ensure old github-universe blog posts redirect to their new url (i.e. `/github-universe/github-universe-liveblog-ceos-keynote`)
